wgsl/index.bs

```diff
@@ -876,22 +996,33 @@ array_decoration
 # Variable and const # {#variables}
 
 TODO: *Stub* (describe what a constant is): A constant is a name for a value, declared via a `const` declaration.
+What types are permitted?  Storable, plus pointer to store type.
+
+TODO(dneto): A const may not be of type pointer-to-handle. A function parameter may not have type pointer-to-handle.
+Otherwise we'd have a need to make a pointer-to-handle type expression. But we've reserved the `handle` keyword.
```

Comment:
Can this be solved by allowing `ptr<texture_type|sampler>` as a type? Which then has the implicit `handle` storage class?

Reply:
I suppose we could, but then that's an irregularity.